posix.mak

```diff
@@ -646,9 +646,9 @@ betterc: betterc-phobos-tests
 	@# Due to the FORCE rule on druntime, make will always try to rebuild Phobos (even as an order-only dependency)
 	@# However, we still need to ensure that the test_extractor is built once
 	@[ -f "$(TESTS_EXTRACTOR)" ] || ${MAKE} -f posix.mak "$(TESTS_EXTRACTOR)"
-	@$(TESTS_EXTRACTOR) --betterC --attributes betterC \
+	$(TESTS_EXTRACTOR) --betterC --attributes betterC \
```

Comment:
IIRC all other executions runs silently in this Makefile.

Reply:
Not all, there many things that are not, e.g.:
https://github.com/dlang/phobos/blob/0243abe6dff323f8fa98ff79de15ac00a66d677a/posix.mak#L400

The primary motivation for me was that I found it hard to see what's going on at first (even though I knew in principle what to expect) and I presume that by enabling slightly more verbose output I would make it easier for next person working on this code.

What I previously saw in the log was this:
```
...
+ make -f posix.mak -j2 betterc
mkdir -p generated/linux/release/64/betterctests || exists generated/linux/release/64/betterctests
touch generated/linux/release/64/betterctests/.directory
parsing std/array.d
parsing std/ascii.d
parsing std/base64.d
parsing std/bigint.d
parsing std/bitmanip.d
parsing std/compiler.d
parsing std/complex.d
parsing std/concurrency.d
parsing std/conv.d
parsing std/csv.d
parsing std/demangle.d
parsing std/encoding.d
parsing std/exception.d
parsing std/file.d
parsing std/format.d
...
parsing std/regex/internal/ir.d
std/typecons.d(1196): Error: `TypeInfo` cannot be used with -betterC
posix.mak:646: recipe for target 'std/range/package.betterc' failed
```

Which didn't make much sense.